nav2_bringup/nav2_gazebo_spawner/nav2_gazebo_spawner/nav2_gazebo_spawner.py

```diff
@@ -78,8 +78,7 @@ def main():
     tree = ET.parse(sdf_file_path)
     root = tree.getroot()
     for plugin in root.iter('plugin'):
-        # TODO(orduno) Handle case if an sdf file from non-turtlebot is provided
-        if 'turtlebot3_diff_drive' in plugin.attrib.values():
+        if 'libgazebo_ros_diff_drive.so' == plugin.get('filename'):
```

Comment:
use `in` instead and just check for `ros_diff_drive` in case those names change again.